File: 168-ExcelSheetColumnTitle/168-ExcelSheetColumnTitle.java
// Last updated: 8/11/2026, 12:24:49 PM
class Solution {
    public String convertToTitle(int n) {
        StringBuilder ans=new StringBuilder();
        while(n>0){
            n--;
            int r=n%26;
            ans.append((char)(r+'A'));
            n/=26;

        }
        return ans.reverse().toString();
    }
}

File: 168-ExcelSheetColumnTitle/168-ExcelSheetColumnTitle.py
# Last updated: 9/2/2026, 12:41:23 PM
class Solution:
    def convertToTitle(self, columnNumber):
        result = ""
        
        while columnNumber:
            columnNumber -= 1
            result = chr(columnNumber % 26 + ord('A')) + result
            columnNumber //= 26
        
        return result